cdc/processor/pipeline/sink.go

```diff
@@ -392,20 +392,26 @@ func (n *sinkNode) closeTableSink(ctx context.Context) (err error) {
 			return
 		}
 		log.Info("sinkV1 is closed",
-			zap.Int64("tableID", n.tableID),
 			zap.String("namespace", n.changefeed.Namespace),
-			zap.String("changefeed", n.changefeed.ID))
+			zap.String("changefeed", n.changefeed.ID),
+			zap.Int64("tableID", n.tableID))
 		err = cerror.ErrTableProcessorStoppedSafely.GenWithStackByArgs()
 		return
 	}
-	err = n.sinkV2.Close(ctx)
-	if err != nil {
-		return
+
+	if err := n.sinkV2.Close(ctx); err != nil {
+		switch errors.Cause(err) {
+		case context.Canceled, context.DeadlineExceeded:
+			// ignore context canceled error
+		default:
```

Comment:
How about move `log.Info("sinkV2 is closed")` here? I worry about other guys can make mistake.